static/css/footer.less

```diff
@@ -100,6 +100,7 @@
 
         &-label {
             display: block;
+            font-size: unit(16px / @base-font-size-px, em);
```

Comment:
[`@base-font-size-px` is `16px`](https://github.com/cfpb/cf-core/blob/gh-pages/src/cf-vars.less#L49), could you say `1em` here? Or would that be problematic if `@base-font-size-px` changed?